lib/active_merchant/billing/gateways/realex.rb

```diff
@@ -43,6 +43,7 @@ class RealexGateway < Gateway
       def initialize(options = {})
         requires!(options, :login, :password)
         options[:refund_hash] = Digest::SHA1.hexdigest(options[:rebate_secret]) if options.has_key?(:rebate_secret)
+        options[:credit_hash] = Digest::SHA1.hexdigest(options[:refund_secret]) if options.has_key?(:refund_secret)
```

Comment:
I think the inconsistencies that we discussed between the use of the terms refund/rebate/credit at Spreedly and Realex makes the naming here unavoidably confusing. I scratched my head a bit after not looking at this for a few days, trying to remember the logic behind this naming iteration. It would be helpful to have a comment here so that someone looking at this code a year down the road isn't confused by ```refund_secret``` being in ```credit_hash``` rather than ```refund_hash```, which seems counterintuitive. Can you please add one?

Reply:
@jasonxp I see what you're saying about the terminology being confusing, but I'm unsure of what sort of comment I can add to help with the confusion beyond saying "yep, it's weird, but it's right". This is my draft of a comment, but I'm not sure if it's what you're looking for or whether it would actually be helpful. 

```
        # Although this looks odd, according to the Realex gateway documentation, the `refund_secret` is indeed used by the `credit` action and not the `refund` action.
```

What do you think?

Reply:
I looked at the docs again to make sure I had my head wrapped around this correctly, and I see the symmetry in your revised naming. Feel free to leave it as-is without the comment. Thanks for taking a stab at it.